opendit/models/latte/latte_t2v.py

```diff
@@ -1138,6 +1146,29 @@ def forward(
             If `return_dict` is True, an [`~models.transformer_2d.Transformer2DModelOutput`] is returned, otherwise a
             `tuple` where the first element is the sample tensor.
         """
+
+        # 0. Split batch for data parallelism
+        if get_data_parallel_size() > 1:
+            conditional = get_data_parallel_rank() == 0
+            (
+                hidden_states,
+                timestep,
+                encoder_hidden_states,
+                added_cond_kwargs,
+                class_labels,
+                attention_mask,
+                encoder_attention_mask,
+            ) = split_batch(
```

Comment:
its very good to have a batch function, but lets make it global usable. for example:
a1, a2 = batch_func(partial(split_sequence, dim=0), a1, a2)
or things like:
a1, a2 = batch_func(split_sequence, (a1, 0), (a2, 0))
or else you think better

Reply:
Fixed. Refer to utils.utils.batch_func()